src/libs/tools/midi.c

```diff
@@ -43,24 +43,14 @@ const char *name(dt_lib_module_t *self)
 
 dt_view_type_flags_t views(dt_lib_module_t *self)
 {
-  return DT_VIEW_ALL;
```

Comment:
Correct. Previously they were added, invisibly, to the header in each view. Now they're not, so some more optional methods can be dropped since they will never be called.

These modules shouldn't _really_ be a "libs" at all; they should be their own category, like the io modules. But since it looks unlikely that more will be introduced, I'm not spending time setting up a separate initialisation loop for them.